docker-compose.yaml

```diff
@@ -28,6 +28,16 @@ services:
       interval: 5s
       timeout: 5s
       retries: 5
+  frontend:
+    build:
+      context: frontend
+    ports:
+      - "${PBTAR_FRONTEND_PORT:-3000}:80"
```

Comment:
it seems very strange to me to set the external facing port to 3000 and the internal port to 80 (the default http port)... it seems like that should be reversed

Reply:
The internal port is 80 mainly because when I was developing locally (using vite) that was the easiest/ fastest way to spin things up. 

I think in general, I'd actually just like every port to have identical mappings on either side of the container (e.g. internal port == external port always) but that's way outside the scope of this PR

Reply:
now that we have multiple services/containers serving web content, I think it's inevitable that some of them will have to use an explicit port to access them, but I'd rather see the Swagger interface at whatever port it commonly uses than having the main web interface at an alternate port that has to be explicitly added to its URL

Reply:
Please make an issue that explicitly states which port each service should be served at, and we can go from there

Reply:
I would expect something like:
Database = 5432
FastAPI = 8000
React = 80

Reply:
#79